README.md

```diff
@@ -74,3 +74,23 @@ The default `print` parameter value is `true`. It means that in this example the
 
 ### Print as a function
 The `print` parameter can also be a function. Then the `props` and `context` are given to it (in this order), and it should return a boolean indicating wether or not the actual component should be displayed.
+
+### Error handling
+The `error` parameter allow to specify a prop that indicates wether or not a placeholder error component should be displayed in replacement of the real component.
+It's usefull when data that are mandatory for the correct display of a component are missing.
+
+Like for the `print` prop, `error` can be a `boolean`, a `string` (referencing a prop name), an array of `string` (an array of prop names) or a `function` (whose result will be converted to `boolean`).
```

Comment:
`As the ``print`` prop` ?

Reply:
Are you sure about this one ? English is not my strong suit however I think using "as" would mean that `error` could be use in place of `print`.

Reply:
ok